File: 0066-plus-one/0066-plus-one.js
/**
 * @param {number[]} digits
 * @return {number[]}
 */
var plusOne = function(digits) {
    let num =digits.join("");
    let newnum =BigInt(num);
    newnum ++;
    let numArr =newnum.toString().split("").map(Number);
    return numArr;
};

File: 0066-plus-one/0066-plus-one.cs
public class Solution {
    public int[] PlusOne(int[] digits) {
        int number =0;
        foreach(int digit in digits){
            number=number*10+digit;
        }
         number+=1;
        int[] result=number
        .ToString()
        .Select(x=>x-'0')
        .ToArray();
        return result;
    }
}